x-pack/plugins/fleet/server/types/models/agent_policy.ts

```diff
@@ -84,6 +84,14 @@ export const AgentPolicyBaseSchema = {
   ),
   ...getSettingsAPISchema('AGENT_POLICY_ADVANCED_SETTINGS'),
   supports_agentless: schema.maybe(schema.boolean({ defaultValue: false })),
+  global_data_tags: schema.maybe(
```

Comment:
should be added to openapi spec too https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/common/openapi/README.md